### PR TITLE
Update dora.py

### PR DESCRIPTION
**Pull request by l4b4r4b4b4**
added missing dataloader in continue_learning

### CODE REVIEW

**Review by catid — APPROVED** [review]

**Comment by catid** [comment]
I think the branch doesn't produce quantifiable results on this simple dataset without SGD.  Maybe conclusions could be drawn with larger models with larger datasets.

**Comment by l4b4r4b4b4** [comment]
I was actually gonna try to build on top of your reference implementation using an actual LLM, DPO datasets from HuggingFace and cosine similarity loss function of generated answer with chosen / rejected respectively. Are you guys somehow affiliated to the actual Paper btw? :thinking:
